vcd/validate_funcs.go

```diff
@@ -112,3 +112,24 @@ func validateMultipleOf4() schema.SchemaValidateFunc {
 		return
 	}
 }
+
+// IntLeaseSeconds validates amount of seconds for lease
+// A value of 0 is accepted, as it means "never expires"
+// Regular values must be > 3600
+func IntLeaseSeconds() schema.SchemaValidateFunc {
+	return func(i interface{}, k string) (s []string, es []error) {
+		v, ok := i.(int)
+		if !ok {
+			es = append(es, fmt.Errorf("expected type of %s to be int", k))
+			return
+		}
+
+		valid := i == 0 || v >= 3600
```

Comment:
I am not use `valid` here is needed. I think it's ok to check directly in condition

Reply:
The current condition check seems to me more readable than

```
if !(i == 0 || v >= 3600) {
```

Reply:
For it isn't, but we can live with that.